frontend/www/js/omegaup/course/edit.js

```diff
@@ -4,9 +4,11 @@ import course_AssignmentDetails from '../components/course/AssignmentDetails.vue
 import course_AssignmentList from '../components/course/AssignmentList.vue';
 import course_Form from '../components/course/Form.vue';
 import course_ProblemList from '../components/course/ProblemList.vue';
+import common_Publish from '../components/common/Publish.vue';
 import course_Clone from '../components/course/Clone.vue';
 import { OmegaUp } from '../omegaup';
-import API from '../api.js';
+import api from '../api';
```

Comment:
```suggestion
import api from '../api_transitional';
```

oops se me fue esto.

Reply:
Aún me es un poco confuso cuando llamar un archivo y cuando llamar el otro. Pero ya lo cambié. La forma correcta de mandarlo llamar es:
```
import * as api from '../api_transitional';
```
de otra forma me mandaba error.

Reply:
usa `api_transitional` en los siguientes casos:

1. siempre que estés introduciendo código nuevo (porque idealmente estarías probando esto a mano y notarías si algo está roto)
2. si estás cambiando código y puedes probar que la versión de `api_transitional` funciona manualmente
3. si la declaración de esa API en `api.js` simplemente redirecciona al mismo método que viene de `api_transitional`.

La razón de esto es que _aún_ no termino el cambio que permite que **todos** los resultados que usan fechas manden los tiempos como `Date` (más bien, que se usen las anotaciones de tipos correctas para que se pueda autogenerar el código de conversión de fechas). Espero que para antes del domingo siguiente logre suficiente progreso como para hacer que `api_transitional` se use por default siempre!